src/modal/_pera-wallet-modal.scss

```
@@ -53,39 +60,104 @@
 }
 
 .pera-wallet-connect-modal__close-button {
-  width: 30px;
-  height: 30px;
+  width: 40px;
+  height: 40px;
 
   margin: 0;
   padding: 0;
 
-  box-shadow: 1px 2px 2px rgba(0, 0, 0, 0.08);
-  background-color: white;
+  background: #333333;
+  border: 1.5px solid rgba(255, 255, 255, 0.08);
+  box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.15);
 
   border-radius: 100%;
 }
 
 .pera-wallet-connect-modal__body {
   position: relative;
 
-  width: 440px;
+  width: 655px;
   max-width: calc(100vw - 80px);
 
-  padding: 24px;
+  padding: 28px;
+
+  background-color: #edeffb;
 
-  background-color: white;
   box-shadow: 1px 2px 2px rgba(0, 0, 0, 0.08);
 
-  border-radius: 4px;
+  border-radius: 24px;
+
+  &::before {
```

We can achieve the background pattern with CSS instead of PNG, let's try this

```suggestion
  &::before {
    --background-line: #1e0972 0 1.2px, transparent 0 calc(100% - 1.2px), #1e0972;

    position: absolute;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;

    content: "";

    background-image: linear-gradient(var(--background-line)),
      linear-gradient(90deg, var(--background-line));
    background-size: 116px 116px;

    mix-blend-mode: overlay;

    border-radius: 24px;

    opacity: 0.8;

    pointer-events: none;
  }
```